broadcaster/_backends/kafka.py

```diff
@@ -0,0 +1,51 @@
+import asyncio
+import typing
+from urllib.parse import urlparse
+
+from aiokafka import AIOKafkaConsumer, AIOKafkaProducer
+
+from .._base import Event
+from .base import BroadcastBackend
+
+
+class KafkaBackend(BroadcastBackend):
+    def __init__(self, urls: typing.Union[str, typing.List[str]]):
```

Comment:
*At the moment* we should start with just `url: str` here, since we don't have any public interface to support passing multiple URLs. However we should also open up an issue to support that.

Reply:
Makes sense, this is pre-alpha so we can still change the interface as needed. I'll open some issues